### DISCUSSION TITLE
plonk-intro-cn/plonk-permutation

### DISCUSSION THREAD

**Discussion by giscus** [z2o-k7e/z2o-k7e.github.io#36] [Announcements]
# plonk-intro-cn/plonk-permutation

零知世界, Halo2 Tutorial, world of Zero-knowledge proof, Halo2 教程

https://learn.z2o-k7e.world/plonk-intro-cn/plonk-permutation.html

**Comment by punish-yh** [comment]
“从 Multiset 等价到置换证明”一节中描述“我们再使用一个技巧：再向 Verifier 索取一个随机数 β，把一个元组「折叠」成一个值”。这里使用β的作用是什么？不使用β可不可以呢？
